### ISSUE TITLE
Suggest not using super large font for password.

### ISSUE THREAD

**Issue by parkerjgit** [pglombardo/PasswordPusher#58]
To prevent someone from reading it from across the room. Also, blur effect is not quite blurry enough to prevent reading. Perhaps have copy to clipboard option so no need to display at all.

**Comment by pglombardo** [comment]
Definitely a fair point.  At a minimum, I would love to add this as a password or default option.

**Comment by alexandrezia** [comment]
I agree, I use to push very large passwords and the line breaks, some users only copy the first line thus not getting the full password

**Comment by Viajaz** [comment]
Could relate to #486 regarding the ability to customise the password field.
